unittest/Dialect/TritonGPU/LinearLayoutConversionsTest.cpp

```diff
@@ -2835,10 +2835,10 @@ TEST_F(LinearLayoutConversionsTest, LeadingOffset_64x4x32_1_8_32b_transposed) {
                       {8, 0, 16},
                       {0, 1, 0},
                       {0, 2, 0},
-                      {16, 0, 0},
-                      {32, 0, 0},
                       {0, 0, 1},
-                      {0, 0, 2}}},
+                      {0, 0, 2},
+                      {16, 0, 0},
+                      {32, 0, 0}}},
```

Comment:
is this change expected? Does that mean the layout used to be wrong?

Reply:
What's happening is the (64x4x32) shape is being split into 4 tma blocks of size (16x4x32) due to the 64-byte swizzling and the tma message dimension is always distributed last. The previous layout is valid for matmul, but not compatible with TMA.